quickplot_instructions.md

```diff
@@ -0,0 +1,122 @@
+- **runs** is the run range in the form "run:ls-run:ls,run:ls[...]" (default: 0:min-999999:max - all runs).
+- **dataset**<sup>(\*\*)</sup> is the input dataset. The code will fetch the input files from DAS. If the *inputFiles* argument is passed, it will be used instead (default: ''). The */StreamExpress/Run\*/ALCARECO* and */ZeroBias/Run*\*/RAW* datasets can be used.
+- **storePrepend** in case files should be fetched from a specific store (default: '' - auto)
+- **highPurity** decides whether to use a high purity track selection (default: false)
+- **tracks** is the input track collection, which depends on the type of input dataset (default: generalTracks)
+- **updateGT** is used to update records, tags and labels in the global tag. Defined in the form "record1:tag1:label1,record2:tag2:label2,[...]".
+- **saveRootFile** decides whether to save the fit results to a *.root* file. Usually needed only for debugging.
+
+*<sup>(\*)</sup>&nbsp;&nbsp;&nbsp;the argument is mandatory*  
+*<sup>(\*\*)</sup>&nbsp;the argument or --inputFiles is mandatory*  
+
+The result of the fit will be a *.txt* file containing the list of runs and the fit results in some format **not** compatible with the CMS database (see below how to convert this file to a *.db* file).  
+If the *saveRootFile* argument is set to *true*, a *.root* file will be created containing the fit results. Both the *.txt* and *.root* file name are set by the *jobName* argument.   
+A file with the "_filelist.txt" prefix is also creted. It contains the list of files used in the fit (**NOTE**: if specific lumisections are analyzed by setting the *runs* argument, the fitter might consume more files than needed as there is no way to fetch single lumisections from DAS. These files will also appear in this list but their content will be skipped).  
+(NOT YET) A file with the "_cfg.py" prefix will be written and will contain the full configuration of the job.
+  
+**NOTE** the track collection from *ALCARECO* is *ALCARECOTkAlMinBias*, for *RAW* the label is *generalTracks*.  
+**NOTE** Other parameters of the fitter are present in the configuration file and usually there is no need to modify them.  
+**NOTE** by default, the fit is run on five lumisections. This can be changed by editing the parameters (note: set them equal).  
+**NOTE** if a track refit is done and the alignment is not changed, the beamspot result will still be different. This is because, in such case, the configuration introduces additional cuts on the PVs.
+**NOTE** the fitter appends the results of the fit to the *.txt* file live, thread safety is **NOT** guaranteed.
+
+## beamspot plot
+
+**NOTE** the plotting script is a work in progress. At the moment it can plot a single file in a format good for check-ups (bs parameter vs. time). For instructions on how to produce complete BS plots, see [here]().  
```

Comment:
```suggestion
**NOTE** the plotting script is a work in progress. At the moment it can plot a single file in a format good for check-ups (bs parameter vs. time).
```